app/views/personal_details/_form.html.erb

```diff
@@ -2,7 +2,7 @@
 
 <div class='govuk-grid-row'>
   <div class='govuk-grid-column-two-thirds'>
-    <%= form_with model: @personal_details, builder: GOVUKDesignSystemFormBuilder::FormBuilder, action: action do |f| %>
+    <%= form_with model: @personal_details, builder: GOVUKDesignSystemFormBuilder::FormBuilder, action: action, html: { novalidate: true } do |f| %>
```

Comment:
👀  I think we accidentally left `html: { novalidate: true }` in as we used it for testing. It'll also be added in [another PR](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/124) so I think we can safely remove this.